fleet/fleet.go

```diff
@@ -217,3 +256,6 @@ func (f fleet) ipFromUnitState(unitState *schema.UnitState) (net.IP, error) {
 
 	return nil, maskAny(ipNotFoundError)
 }
+
+// TODO
```

Comment:
It operates on the fleet unit status. The package defines the status and knows how to deal with it. This was my thought. Putting it somewhere else we spread details and knowledge about a package's implementation across the whole project. The caller definitely must be the CLI command side. Functionality dealing with the fleet status should be within the fleet package, IMO. 

What do I miss?